src/transformers/training_args.py

```diff
@@ -1357,11 +1358,22 @@ def __post_init__(self):
                 if self.use_cpu and not is_torch_bf16_cpu_available() and not is_torch_tpu_available():
                     # cpu
                     raise ValueError("Your setup doesn't support bf16/(cpu, tpu, neuroncore). You need torch>=1.10")
-                elif not self.use_cpu and torch.cuda.is_available() and not is_torch_bf16_gpu_available():
-                    # gpu
-                    raise ValueError(
-                        "Your setup doesn't support bf16/gpu. You need torch>=1.10, using Ampere GPU with cuda>=11.0"
-                    )
+                elif not self.use_cpu:
+                    if torch.cuda.is_available() and not is_torch_bf16_gpu_available():
+                        # gpu
+                        raise ValueError(
+                            "Your setup doesn't support bf16/gpu. You need torch>=1.10, using Ampere GPU with cuda>=11.0"
+                        )
+                    elif not is_torch_xpu_available():
+                        # xpu
+                        import torch
+
+                        parsed_torch_version_base = version.parse(version.parse(torch.__version__).base_version)
+                        is_torch_greater_or_equal_than_1_12 = parsed_torch_version_base >= version.parse("1.12")
```

Comment:
`is_torch_greater_or_equal_than_1_12` is already defined in pytorch_utils and should be imported from there

```suggestion
```